roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_predicates.py

```diff
@@ -52,32 +52,34 @@ def run(self, terms, variables=None, regions_enabled=True, short_version=None,
             # convert short_version to origin short_version
             short_version = re.sub('^3.', '1.', short_version)
 
+        # Predicates ordered according to OpenShift Origin source:
+        # origin/vendor/k8s.io/kubernetes/plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go
+
+        # Default predicates invoked from GeneralPredicates
         if short_version in ['1.1', '1.2']:
             predicates.append({'name': 'PodFitsHostPorts'})
             predicates.append({'name': 'PodFitsResources'})
-
-        # applies to all known versions
-        predicates.append({'name': 'NoDiskConflict'})
```

Comment:
Looks like you are re-ordering the predicates for all versions, instead of just the latest version...